package.json

```diff
@@ -77,6 +77,7 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-vue": "^7.13.0",
     "eslint-plugin-vuejs-accessibility": "^0.7.1",
+    "flush-promises": "^1.0.2",
```

Comment:
this package helps to flush all promises which means that we don't need to use `wrapper.vm.$nextTick()` twice or more in order to wait for all promises to get resolved.

should we use it?

Reply:
@dpakach does this helps for the test you're writing for autocomple inputs taking some time to trigger. If such problems with static waits or of course multiple nextTick can be resolved...imo this might be useful

Reply:
sadly doesn't works in my case

Reply:
This PR needs a rebase and I'm not sure why there was an attempt to use promises so heavily here that it needs an additional library to work around it. Trying a different approach, I'll ping you once I know if it works;)

Reply:
@pascalwengerter, @kiranparajuli589 has made an update to this PR. Please review.